cli/test/e2e/login-key.e2e-spec.ts

```diff
@@ -39,4 +49,27 @@ describe(`login-key (e2e)`, () => {
   it('should log in when providing the correct API key', async () => {
     await new LoginCommand(CLI_BASE_OPTIONS).run(instanceUrl, apiKey);
   });
+
+  it('should create an auth file when logging in', async () => {
+    await new LoginCommand(CLI_BASE_OPTIONS).run(instanceUrl, apiKey);
+
+    const data: string = await readFile(TEST_AUTH_FILE, 'utf8');
+    const parsedConfig = yaml.parse(data);
+
+    expect(parsedConfig).toEqual(expect.objectContaining({ instanceUrl, apiKey }));
+  });
+
+  it('should create an auth file with chmod 600', async () => {
+    await new LoginCommand(CLI_BASE_OPTIONS).run(instanceUrl, apiKey);
+
+    const data = await stat(TEST_AUTH_FILE);
+
+    expect(convert.object(data.mode)).toEqual(
```

Comment:
maybe we could check the permissions without adding a new dependency? I just did a quick google and it looks pretty straightforward: https://www.martin-brennan.com/nodejs-file-permissions-fstat/

Reply:
done, thanks